configs/faster_rcnn/faster_rcnn_r50_caffe_fpn_mstrain_90000it_coco.py

```diff
@@ -0,0 +1,16 @@
+_base_ = 'faster_rcnn_r50_caffe_fpn_mstrain_1x_coco.py'
+
+# learning policy
+lr_config = dict(
+    policy='step',
+    warmup='linear',
+    warmup_iters=500,
+    warmup_ratio=0.001,
+    step=[60000, 80000])
+total_epochs = 90000
+
+# Runner type
+runner = 'IterBasedRunner'
+
+checkpoint_config = dict(interval=10000)
+evaluation = dict(interval=10000, metric='bbox')
```

Comment:
The configuration of runner should be a dict, see examples in [mmcls here](https://github.com/open-mmlab/mmclassification/blob/master/configs/_base_/schedules/imagenet_bs2048.py#L12).